From bef92c34a367fdeb6ffb043232645207984e177d Mon Sep 17 00:00:00 2001
From: "kangsik81.kim" <kangsik81.kim@samsung.com>
Date: Fri, 28 Jun 2013 13:24:59 +0900
Subject: [PATCH] JQM: Remove scrollTo method.

This patch is made to prevent a symptoms that is showing browser's scrollbar when webapp launched.
---
 .../jquery-mobile-1.2.0/js/jquery.mobile.init.js   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/js/jquery-mobile-1.2.0/js/jquery.mobile.init.js b/libs/js/jquery-mobile-1.2.0/js/jquery.mobile.init.js
index d31ddf0..5426059 100644
--- a/libs/js/jquery-mobile-1.2.0/js/jquery.mobile.init.js
+++ b/libs/js/jquery-mobile-1.2.0/js/jquery.mobile.init.js
@@ -122,7 +122,7 @@ define( [ "jquery", "./jquery.mobile.core", "./jquery.mobile.support", "./jquery
 	// then check what the scroll top is. Android will report 0... others 1
 	// note that this initial scroll won't hide the address bar. It's just for the check.
 	$(function() {
-		window.scrollTo( 0, 1 );
+		// window.scrollTo( 0, 1 );
 
 		// if defaultHomeScroll hasn't been set yet, see if scrollTop is 1
 		// it should be 1 in most browsers, but android treats 1 as 0 (for hiding addr bar)
-- 
1.7.9.5

